src/features/zaaksysteem/components/ZaakContactmomenten.vue

```diff
@@ -78,11 +78,11 @@ table {
   }
 
   tr {
-    display: flex;
     padding: var(--spacing-default);
 
     & > * {
-      flex: 1;
+      width: 25%;
```

Comment:
If this is the layout you wanted, that's fine. If you just didn't want to break the dates to a new line, `white-space: nowrap;` would have done the trick too.